src/include/catalog/postgres/pg_proc.h

```diff
@@ -71,19 +71,24 @@ constexpr proc_oid_t SIN_PRO_OID = proc_oid_t(89);
 // TODO(tanujnay112) This overflows into the next internal oid range and will continue to do so
 constexpr proc_oid_t TAN_PRO_OID = proc_oid_t(90);
 constexpr proc_oid_t COT_PRO_OID = proc_oid_t(91);
-constexpr proc_oid_t LOWER_PRO_OID = proc_oid_t(92);
-constexpr proc_oid_t UPPER_PRO_OID = proc_oid_t(93);
-constexpr proc_oid_t COSH_PRO_OID = proc_oid_t(127);
-constexpr proc_oid_t SINH_PRO_OID = proc_oid_t(128);
-constexpr proc_oid_t TANH_PRO_OID = proc_oid_t(129);
-constexpr proc_oid_t VERSION_PRO_OID = proc_oid_t(150);
-
-constexpr proc_oid_t NP_RUNNERS_EMIT_INT_PRO_OID = proc_oid_t(94);
-constexpr proc_oid_t NP_RUNNERS_EMIT_REAL_PRO_OID = proc_oid_t(95);
-constexpr proc_oid_t NP_RUNNERS_DUMMY_INT_PRO_OID = proc_oid_t(96);
-constexpr proc_oid_t NP_RUNNERS_DUMMY_REAL_PRO_OID = proc_oid_t(97);
+constexpr proc_oid_t COSH_PRO_OID = proc_oid_t(92);
+constexpr proc_oid_t SINH_PRO_OID = proc_oid_t(93);
+constexpr proc_oid_t TANH_PRO_OID = proc_oid_t(94);
+constexpr proc_oid_t CEIL_PRO_OID = proc_oid_t(95);
+constexpr proc_oid_t FLOOR_PRO_OID = proc_oid_t(96);
+constexpr proc_oid_t TRUNCATE_PRO_OID = proc_oid_t(97);
+constexpr proc_oid_t LOG10_PRO_OID = proc_oid_t(98);
+constexpr proc_oid_t LOG2_PRO_OID = proc_oid_t(99);
+constexpr proc_oid_t LOWER_PRO_OID = proc_oid_t(100);
+constexpr proc_oid_t UPPER_PRO_OID = proc_oid_t(101);
+constexpr proc_oid_t VERSION_PRO_OID = proc_oid_t(102);
+
+constexpr proc_oid_t NP_RUNNERS_EMIT_INT_PRO_OID = proc_oid_t(103);
+constexpr proc_oid_t NP_RUNNERS_EMIT_REAL_PRO_OID = proc_oid_t(104);
+constexpr proc_oid_t NP_RUNNERS_DUMMY_INT_PRO_OID = proc_oid_t(105);
+constexpr proc_oid_t NP_RUNNERS_DUMMY_REAL_PRO_OID = proc_oid_t(106);
 
 // SQL Function
-constexpr proc_oid_t DATE_PART_PRO_OID = proc_oid_t(101);
+constexpr proc_oid_t DATE_PART_PRO_OID = proc_oid_t(107);
```

Comment:
Move with this up with the other SQL functions.

Reply:
Fixed